FightPandemics/Utilities/Navigation/Navigator.swift

```diff
@@ -53,21 +53,16 @@ final class Navigator {
     private func rootTabBarController() -> RootTabBarController {
         let rootTabBarController = UIStoryboard(name: "Main", bundle: nil)
             .instantiateViewController(withIdentifier: "RootTabBarController") as! RootTabBarController
-        let homeTab = rootNavController(from: HomeViewController.self, storyboardName: "Home", localizedTabBarTitleKey: "Home")
-        self.homeNavigationController = homeTab
-        rootTabBarController.viewControllers = [homeTab]
+        let homeNavigationController = rootTabBarController.viewControllers![0] as! UINavigationController
```

Comment:
i had to change the ViewController placements a bit - in order to localize the Tab Bar titles themselves, we need to use a Storyboard with the `UITabBarItem`s defined in it (learned something new!)